server/sonar-web/src/main/js/helpers/measures.js

```diff
@@ -321,7 +322,8 @@ function shortDurationFormatter (value) {
   if (value === 0 || value === '0') {
     return '0';
   }
-  const hoursInDay = window.SS.hoursInDay;
+  // FIXME
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)